Maui.RevenueCat.InAppBilling/Platforms/Android/RevenueCatBillingAndroid.cs

```diff
@@ -14,7 +14,7 @@ namespace Maui.RevenueCat.InAppBilling.Services;
 public partial class RevenueCatBilling : IRevenueCatBilling
 {
     private Purchases _purchases = default!;
-    private List<Package> _cachedOfferingPackages = new();
+    private Offerings _cachedOfferingPackages = null;
```

Comment:
when you put NULL as default value please make `Offerings` explicitly nullable by: `Offerings?`